### PR TITLE
QuestionIntervieweeAnswerEntity has been updated.

### PR DESCRIPTION
**Pull request by CrazyKamenwik**
SessionId and QuestionSetId has been added.

### CODE REVIEW

File: AIS/AIS.DAL/Entities/QuestionIntervieweeAnswerEntity.cs [hunk 1]
```diff
@@ -6,6 +6,8 @@ public class QuestionIntervieweeAnswerEntity
         public string Text { get; set; }
         public int Mark { get; set; }
         public int QuestionId { get; set; }
+        public int QuestionSetId { get; set; }
+        public int SessionId { get; set; }
```
Comment (GagarinPavel) [inline AIS/AIS.DAL/Entities/QuestionIntervieweeAnswerEntity.cs:10]: add Session and QuestionSet fields also and change SessionEntity and QuestionSetEntity ( add         public virtual ICollection<QuestionIntervieweeAnswerEntity> QuestionIntervieweeAnswers { get; set; } )

File: AIS/AIS.DAL/Entities/SessionEntity.cs [hunk 2]
```diff
@@ -15,5 +16,7 @@ public class SessionEntity
         public DateTime FinishedAt { get; set; }
         public int QuestionAreaId { get; set; }
         public virtual QuestionAreaEntity QuestionArea { get; set; }
+        public virtual ICollection<QuestionIntervieweeAnswerEntity> QuestionIntervieweeAnswer { get; set; }
```
Comment (nantonov) [inline AIS/AIS.DAL/Entities/SessionEntity.cs:19]: s

File: AIS/AIS.DAL/Entities/QuestionSetEntity.cs [hunk 3]
```diff
@@ -9,5 +9,6 @@ public class QuestionSetEntity
         public virtual QuestionAreaEntity QuestionArea { get; set; }
         public int QuestionAreaId { get; set; }
         public virtual ICollection<QuestionEntity> Questions { get; set; }
+        public virtual ICollection<QuestionIntervieweeAnswerEntity> QuestionIntervieweeAnswer { get; set; }
```
Comment (nantonov) [inline AIS/AIS.DAL/Entities/QuestionSetEntity.cs:12]: Unnecessary

File: AIS/AIS.DAL/Entities/QuestionIntervieweeAnswerEntity.cs [hunk 4]
```diff
@@ -7,6 +7,10 @@ public class QuestionIntervieweeAnswerEntity
         public int Mark { get; set; }
         public int QuestionId { get; set; }
         public virtual QuestionEntity Question { get; set; }
+        public int SessionId { get; set; }
+        public virtual SessionEntity Session { get; set; }
+        public int QuestionSetId { get; set; }
```
Comment (nantonov) [inline AIS/AIS.DAL/Entities/QuestionIntervieweeAnswerEntity.cs:12]: unnecessary to link to QuestionSet

**Review by nantonov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**